website/blog/2024-05-22-latest-dbt-stability-improvement-innovation.md

```diff
@@ -0,0 +1,120 @@
+
+### Step 4: **Canary deployment**
+
+Once *all* the aforementioned tests have passed, we roll out the latest deployment to a small subset (5%) of accounts, including our own Internal Analytics project.
+
+These "canary" deployments are continually monitored against a set of precise observability metrics. Metrics we monitor include overall job error and cancellation rates to ensure they don't deviate from our expectations relative to a stable baseline. Any anomaly immediately alerts us, and we can shut off the canary in a matter of seconds, before any additional accounts are affected.
+
+### Step 5: Phased **rollout**
+
+Once the canary deployment has been proven to run stably for at least 24 hours, we mark it as eligible for all accounts to upgrade in their next scheduled deployment of dbt Cloud.
+
+<aside>
+ℹ️ Even with a robust testing, deployment, and monitoring system in place, it will never be *impossible* for a breaking change to make it through — just as in any other SaaS application.
+
+If this does happen, we commit to identifying and rolling back any breaking changes as quickly as possible. Under the new testing and deployment model in dbt Cloud, we have consistently demonstrated to have been able roll back changes in less than an hour and ship roll forward bug fixes in less than 4 hours.
+
+All incidents are retrospected to make sure we not only identify and fix the root cause(s), but also promptly put in place testing, automation, and quality gates to ensure that the same problem never happens again.
+
+</aside>
+
+The outcome of this process is that, when you select "Keep on latest version" in dbt Cloud, the time between an improvement or fix being made to dbt Core and you *safely* getting access to it in your projects is a matter of days — rather than months of waiting for the next dbt Core release, on top of any additional time it may have taken to actually carry out the upgrade.
+
+We're pleased to say that since the beta launch of "Keep on latest version" in dbt Cloud, **we have not had any functional regressions reach customers**, while we've also been shipping 5-10 improvements to core dbt functionality every day. This is a foundation that we aim to build on for the foreseeable future.
+
+## Stability as a feature
+
+A rigorous testing pipeline in dbt Cloud is crucial, but real ongoing stability required some deeper changes in the dbt framework. We take our responsibility as the maintainers of dbt Core seriously, as well the open-source ecosystem around it.
+
+We've ******taken a longer release cycle for the upcoming 1.8 release of dbt Core to revisit some of the "do later" design choices we made in the past — specifically around adapter compatibility, behavior change management, and metadata artifacts.
```

Comment:
not sure if we want asterisks here? or are we bolding?
```suggestion
We've taken a longer release cycle for the upcoming 1.8 release of dbt Core to revisit some of the "do later" design choices we made in the past — specifically around adapter compatibility, behavior change management, and metadata artifacts.
```

Reply:
good catch - this is just a weird vestige of notion -> markdown export